MapboxDirections/MBRouteLeg.swift

```diff
@@ -129,10 +134,13 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    open let segmentDistances: [CLLocationDistance]?
+    
+    open let segmentExpectedTravelTimes: [TimeInterval]?
     
-    open let segmentAttributes: [Attribute:[Any]]?
+    open let segmentSpeeds: [CLLocationSpeed]?
     
-    open let nodeAttributes: [Attribute:[Any]]?
+    open let nodeOpenStreetMapNodeIdentifiers: [Int]?
```

Comment:
“Node” is in the name twice, unnecessarily: rename this property to `openStreetMapNodeIdentifiers`.